index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#"><h2>Welcome</h2></a>
+        <img src="images/more.png" alt="more" />
+      </nav>
+    </header>
+    <main>
+      <section class="main-title">
+        <div>
+          <h1 class="main-header">Hey there. I'm Nati</h1>
+          <h1 class="highligh-title">I'm a software developer</h1>
+        </div>
+        <div>
+          <p class="about-me">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+      </section>
+      <ul class="main-icons">
+        <li><img class="icons" src="images/twitter.png" alt="twitter" /></li>
+        <li><img class="icons" src="images/Linkedin.png" alt="Linkedin" /></li>
+        <li><img class="icons" src="images/github.png" alt="github" /></li>
```

Comment:
- [ ] Please wrap each one of these social media images with a `<a>` tag because it's supposed to be links.
for example:
`<li><a href="your link"><img class="icons" src="images/twitter.png" alt="twitter" /></a></li>`